### DISCUSSION TITLE
Discussions

### DISCUSSION THREAD

**Discussion by andrejsharapov** [andrejsharapov/vue-box-shadows#5] [Ideas]
# 👋 Welcome to Vue Box-shadows
You can use the Discussions as a place to connect with other members of our community. You can also join our discord channel.

[![discord][discord-image]][discord]

We hope that you:
>  * Ask questions you’re wondering about.
>  * Share ideas.
>  * Engage with other community members.
>  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

[discord]: https://discord.gg/XtT4Hdf3
[discord-image]: https://img.shields.io/badge/discord-channel-5865f2.svg

**Comment by qorncat** [comment]
pls mark answer and then remove again

**Comment by andrejsharapov** [comment]
Idea: give each element a name instead of numbers

**Comment by andrejsharapov** [comment]
Adapt for vue 3
